server/server/settings.py

```diff
@@ -222,6 +222,7 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#auth-password-validators
 
 AUTHENTICATION_BACKENDS = [
+    "axes.backends.AxesStandaloneBackend",
```

Comment:
See #80. I looked at this and issue #57, and my first thought was that it's something the `phac_helpers` library should be enforcing, since that's where `AxesMiddleware` is added and configured from. My second thought was to check if it already does, and it turns out the answer is yes, as long as the project calls `configure_authentication_backends` haha

Reply:
This was failing (can't remember which build step) without adding this, but there was an error indicating that it needed this, and when adding configure_authentication_backends in, it seemed to resolve the issue (so I added it).  But this is when everything was failing.  

Just removed this line, and tried it again - and you're right it works!  There's just a warning - so keeping this out. 
![image](https://github.com/PHACDataHub/cpho-phase2/assets/38951930/ee5f8c9a-2033-4179-a327-e9ba7a8879ac)